okio/src/main/kotlin/okio/SegmentPool.kt

```diff
@@ -28,14 +25,11 @@ internal object SegmentPool {
   const val MAX_SIZE = 64 * 1024L // 64 KiB.
 
   /** Singly-linked list of segments.  */
-  @JvmField
```

Comment:
Were these annotations unused?

Reply:
It's an internal class, and the code's all Kotlin now, so I don't so how they could have been.